### ISSUE TITLE
HISTORY.md newest position is called (unreleased)

### ISSUE THREAD

**Issue by lipowskm** [rochacbruno/python-project-template#18]
When making a new release, `HISTORY.md` newest position is called `(unreleased)` instead of the newest version.

I've created a simple workaround by modifying `release` under `Makefile`.

Instead of calling:

```bash
@$(ENV_PREFIX)gitchangelog  > HISTORY.md
```

You could perhaps do something like:

```bash
@$(ENV_PREFIX)gitchangelog | sed "s/(unreleased)/$${TAG} ($$(date +%Y-%m-%d))/g" > HISTORY.md
```

Furthermore, you could get rid of all the release commits from changelog:

```bash
@$(ENV_PREFIX)gitchangelog | sed "s/(unreleased)/$${TAG} ($$(date +%Y-%m-%d))/g" | grep -v "Release: version" > HISTORY.md
```

This way `HISTORY.md` contains relevant commits only.

**Comment by kevin-kruse** [comment]
Yeah, got the same question: How do you tag a git commit, but append the history to this very git commit afterwards (with gitchangelog)?
